receiver/awscontainerinsightreceiver/internal/k8sapiserver/k8sapiserver.go

```diff
@@ -560,18 +562,21 @@ func (k *K8sAPIServer) getPersistentVolumeMetrics(clusterName, timestampNs strin
 
 	pvMetrics := k.leaderElection.persistentVolumeClient.GetPersistentVolumeMetrics()
 
-	clusterFields := map[string]any{
+	fields := map[string]any{
 		ci.MetricName(ci.TypePersistentVolume, ci.Count): pvMetrics.ClusterCount,
 	}
 
-	clusterAttributes := map[string]string{
+	attributes := map[string]string{
 		ci.ClusterNameKey: clusterName,
 		ci.MetricType:     ci.TypePersistentVolume,
 		ci.Timestamp:      timestampNs,
 		ci.Version:        "0",
 		ci.SourcesKey:     "[\"apiserver\"]",
 	}
-	md := ci.ConvertToOTLPMetrics(clusterFields, clusterAttributes, k.logger)
+	if k.nodeName != "" {
+		attributes[ci.NodeNameKey] = k.nodeName
+	}
```

Comment:
We do this in enough that it might be worth creating a function that populates the attributes with any defaults.